src/Microsoft.Extensions.ApiDescription.Client/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -235,6 +245,13 @@
   <!-- ServiceFileReference support -->
 
   <Target Name="_ServiceFileReferenceGenerator_GetMetadata" Condition="'@(ServiceFileReference)' != ''">
+    <!-- Repeat default namespace checks in case $(RootNamespace) is defined after .props file is imported. -->
```

Comment:
We shouldn't do things like this, it's not idiomatic MSBuild. If these names are set elsewhere we should remove that and keep this.

Reply:
👍